cogs/status.py

```diff
@@ -1,71 +1,62 @@
 from asyncio import wait_for
-from discord.ext import commands, bridge
-from discord.ext.bridge import Bot
-from discord import Embed, utils as dutils
-from utils import Utils
+from discord import slash_command, option, Embed, utils as dutils
+from discord.ext import commands
+from discord.ext.commands import Bot
+from utilities.data import remove_colors_from_string, Colors, get_server_status
+from utilities.database import selector
+from utilities.utility import check_ip
+
 
 class Status(commands.Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
-    @bridge.bridge_command(aliases=["s"], description="Get the server status")
+    @slash_command(aliases=["s"], description="Get the server status")
+    @option("serverip", str, description="The Aternos-IP to check")
     async def status(self, ctx, serverip=None):
         if serverip is None:
-            cursor = await Utils.mysql_login()
-            database = cursor.cursor()
-            database.execute("SELECT server_ip FROM server WHERE guild_id = %s", [ctx.guild.id])
-            try:
-                result = database.fetchone()[0]
-            except TypeError:
-                database.close()
-                cursor.close()
-                return await Utils.respond(ctx, "Sorry, but this server does not have an IP registered. Please use `setserver` for that.")
-            serverip = result
-            database.close()
-            cursor.close()
-        if not serverip.endswith(".aternos.me"):
-            serverip += ".aternos.me"
-        if serverip.count(".") > 2:
-            return await Utils.respond(ctx, "Please provide a valid Aternos server ip!\nExample: example.aternos.me")
-        if serverip.count(":") == 1:
-            if len(serverip.split(":")[1]) != 5:
-                return await Utils.respond(ctx, "Please provide a valid Aternos server ip!\nExample: example.aternos.me")
+            serverip = (await selector('SELECT server_ip FROM server WHERE guild_id = %s', [ctx.guild.id]))[0]
+            if not serverip:
+                return await ctx.respond("Sorry, but this server does not have an IP registered. Please use `setserver` for that.")
+        serverip = check_ip(serverip)
+        if not serverip:
+            return await ctx.respond("Please provide a valid Aternos IP.")
         await ctx.defer()
         try:
-            stat = await wait_for(Utils.get_server_status(serverip), timeout=3)
+            stat = await wait_for(get_server_status(serverip), timeout=3)
         except TimeoutError:
-            return await Utils.respond(ctx, "Uh oh! The protocol took too long to respond! This will likely fix itself.")
+            return await ctx.respond("Uh oh! The protocol took too long to respond! This will likely fix itself.")
```

Comment:
```suggestion
            return await ctx.respond("Uh oh! The protocol took too long to respond! This will likely fix itself.", ephemeral=True)
```